app/models/target.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: targets
+#
+#  id         :bigint           not null, primary key
+#  latitude   :float            not null
+#  longitude  :float            not null
+#  radius     :float            not null
+#  title      :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  topic_id   :bigint           not null
+#  user_id    :bigint           not null
+#
+# Indexes
+#
+#  index_targets_on_topic_id  (topic_id)
+#  index_targets_on_user_id   (user_id)
+#
+class Target < ApplicationRecord
+  belongs_to :user
+  belongs_to :topic
+
+  validates :title, :latitude, :longitude, :radius, :user, :topic, presence: true
```

Comment:
As the target belongs to the user, you don't need to validate the user presence, rails will do it by default. Unless optional flag is present in the association, e.g: belongs_to :user, optional: true